projects/azure/azure-cicd/script-cm.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: token-refresh-script
+data:
+  entrypoint.sh: |-
+    #!/bin/bash
+    export KUBECONFIG=/tmp/registry
```

Comment:
I don't see you mounting a kubeconfig in the cronjob. Also why not simply use the in-cluster sa token instead?

Reply:
we dont need to mount it. Export acts as placeholder where to write config when doing oc login.

Flow is:
mount non-expiring password with secret `password`.
set temp kubeconfig location
login with non-expiring user to aws registry. This will generate 30d token.
Set back to ci-cd cluster kubeconfig
Update 30d secret with temporary token to be used for rhel builds.
---
Secret is read by pre-prelease RHEL builds and passed to openshift-ansible
Openshift ansible backed this token into rhel images for 30d.
Profit 

All this is part of 3 PR together: https://trello.com/c/fn4hnPT4/173-3-images-build-rhel-310-image-for-hcp-testing  and will be completed/tested when Steve adds kubeconfig I sent him on top of your fridays email.

Reply:
Add the flow as a comment in this file.